test/test.js

```diff
@@ -1205,6 +1212,15 @@ describe('Page', function() {
       expect(response.status()).toBe(200);
       expect(response.url()).toContain('self-request.html');
     });
+    // TODO edit to vefify that results are retrieved
+    it('should support ignoreHTTPSErrors option', async({page_https, httpsServer}) => {
```

Comment:
There are other ignoreHTTPErrors tests; let's move this one next to those. This way you'll avoid spawning second browser for all page tests.

Reply:
The only other test with `ignoreHTTPSErrors` I can spot is on line 111 testing the whole functionality of the response. Did you mean this occurrence?